terraform/modules/task-definitions/publisher/main.tf

```diff
@@ -63,6 +63,142 @@ data "aws_secretsmanager_secret" "sentry_dsn" {
 
 locals {
   service_name = "publisher"
+  # subdomain = join(compact([local.service_name, var.procfile_process]), "-")
```

Comment:
Just in case you uncomment this at some later point, the parameters for [`join()`](https://www.terraform.io/docs/configuration/functions/join.html) are the other way around i.e. `join(separator, list)`